src/platform_impl/macos/view.rs

```diff
@@ -1126,3 +1136,35 @@ fn replace_event(event: &NSEvent, option_as_alt: OptionAsAlt) -> Id<NSEvent> {
         event.copy()
     }
 }
+
+// Convert given NSRange to Rust String Range data used in this crate.
+fn get_actual_range_bounds(source_str: &str, range: NSRange) -> Option<(usize, usize)> {
+    // The given NSRange length may be 0 to indicate a cursor in lieu of a marked range.
```

Comment:
```suggestion
    // The given NSRange length may be 0 to indicate a cursor in line of a marked range.
```

typo?

Reply:
Google it. I learned this expression from Dr. Ken Lunde.